cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -262,6 +367,15 @@ cros_gralloc_driver::cros_gralloc_driver()
 			drv_kms_ = drv_render_;
 		}
 
+		if (ivshm_node_idx != -1) {
+			if (!(drv_ivshmem_ = drv_create(node_fd[ivshm_node_idx]))) {
+				drv_loge("Failed to create driver for the ivshm device with card id %d\n",
+					ivshm_node_idx);
+				close(node_fd[ivshm_node_idx]);
+			} else
+				gpu_grp_type = GPU_WITH_IVSHMEM;
```

Comment:
The other value may be overlapped here, which will impact i915 code below. And GPU_WITH_IVSHMEM is not used anywhere.
		if (((drv->gpu_grp_type == TWO_GPU_IGPU_DGPU) ||
		    (drv->gpu_grp_type == THREE_GPU_IGPU_VIRTIO_DGPU))
			&& (i915->graphics_version >= 125))
			return 0;

Reply:
Just thought that the gpu scenario is more complicate, i add some gpu type, please check.